mobile-app/app/screens/AppNavigator/screens/Settings/screens/FeatureFlagScreen.tsx

```diff
@@ -0,0 +1,130 @@
+import { ThemedScrollView, ThemedText, ThemedView } from '@components/themed'
+import { Switch } from '@components/index'
+import { tailwind } from '@tailwind'
+import { translate } from '@translations'
+import * as React from 'react'
+import { useEffect, useState } from 'react'
+import { View } from 'react-native'
+import { FeatureFlag, FEATURE_FLAG_ID } from '@shared-types/website'
+import { useFeatureFlagContext } from '@contexts/FeatureFlagContext'
+import { WalletAlert } from '@components/WalletAlert'
+
+export interface BetaFeaturesI extends FeatureFlag {
+  value: boolean
+}
+
+export function FeatureFlagScreen (): JSX.Element {
+  const { featureFlags, enabledFeatures, updateEnabledFeatures } = useFeatureFlagContext()
+  const features = featureFlags.filter((item) => item.stage === 'beta')
+  const [betaFeatures, setBetaFeatures] = useState<BetaFeaturesI []>([])
+
+  const getBetaFeature = (flags: FEATURE_FLAG_ID[]): BetaFeaturesI[] => {
+    return features.map((item: FeatureFlag) => {
```

Comment:
we are looping here twice (loop on another loop). Maybe we can change it to a better option